Python/ceval.c

```diff
@@ -1064,9 +1064,11 @@ _PyEval_EvalFrameDefault(PyFrameObject *f, int throwflag)
            Py_MakePendingCalls() above. */
 
         if (_Py_atomic_load_relaxed(&eval_breaker)) {
-            if (_Py_OPCODE(*next_instr) == SETUP_FINALLY) {
+            if (_Py_OPCODE(*next_instr) == SETUP_FINALLY
+                || _Py_OPCODE(*next_instr) == YIELD_FROM) {
```

Comment:
What if the function had some variable it creates before it calls ``yield from`` on it?

like for example:

```py
# snip
    def generator1(self):
        something = None
        return (yield from self.generator2())

# snip
```